quality-assurance/user-acceptance-testing.md

```diff
@@ -0,0 +1,41 @@
+## What is User Acceptance Testing
+
+UAT means testing software against its business requirements i.e. does it solve the business problem as intended?
+
+## Who carries out UAT
+
+Business users are best placed to do UAT, as they are aware of the reality of their business, business processes, roles etc.  Sometimes, a tester will act in place of users.
+
+## When does UAT happen
+
+It will typically take place once a functioning system has been delivered, often after system testing.
+
+## Is UAT always necessary
+
+No.  If a project been delivered with an agile process, each increment of the software should have shown to its users during development.  In this case UAT might not be necessary.
+
+UAT is most likely to be required when software has been delivered by external partner following a waterfall process.
```

Comment:
the comercialls need to spell out with no ambiguity of what artifacts need to be delivered for the waterfall approach if the supplier is deemed incapable of working in an effective agile manor.
eg unit tests delivered and build and run independently.
integration /system and ui tests all build and run independent with data that can be seeded by ukho or third party
timelines and entry criteria for the UAT stage is clear

Reply:
added statement about SoR